src/app/components/Text/index.jsx

```diff
@@ -36,29 +54,11 @@ Bold.propTypes = {
 
 Text.propTypes = {
   text: string.isRequired,
-  options: shape({
-    forceBlock: bool,
-  }),
+  paragraphOverride: func,
```

Comment:
paragraphOverride was previously set as a node which caused the console to error. It expected a function due to the typeOf Styledparagraph, which paragraphOverride inherits from, being a function.